guides/v2.2/release-notes/ReleaseNotes2.2.6CE.md

```diff
@@ -1677,13 +1677,16 @@ You can find Magento Shipping-specific release notes in [Magento Shipping Releas
 
 
 
-## Known issue
+## Known issues
 
-The `catalog:image:resize` command execution time has been reduced by up to 90% in the release. However, this improvement necessitates these additional steps after upgrading your Magento instance to 2.2.6: 
+**Issue**: Merchants cannot save a newly created multiselect or dropdown customer attribute or edit existing customer attributes from the customer’s account on the storefront after upgrade to Magento 2.2.6. **Workaround**: Download and apply hot fix [MAGETWO-95591](https://magento.com/tech-resources/download#download2248). This issue will be fixed in Magento Open Source 2.2.7, which is scheduled for release by the end of 2018. 
```

Comment:
Should the workaround be on it's own line, to highlight the workaround? I like the added issue and workaround, the info as to the issue and fix, etc.